types/events.ts

```diff
@@ -415,6 +415,7 @@ export type ReceivedPaymentCreated = BaseEvent & {
         status: ReceivedPaymentStatus
         type: string
         amount: number
+        isAdvaceable: boolean
```

Comment:
isAdvanceable

Reply:
@axshani  good call
unfortunately, i copy pasted this from the response i got to a `GET /received-payments` request so pretty sure you have a typo somewhere else in the code